internal/pkg/template/diff/diff.go

```diff
@@ -124,28 +147,43 @@ func parseSequence(fromNode, toNode *yaml.Node) (map[string]*Node, error) {
 			if diff.err != nil {
 				return nil, diff.err
 			}
-			children[nextKey()] = diff.node
+			children = append(children, &seqItemNode{
+				node: node{
```

Comment:
nit: why is the syntax different here than the rest of the case statements?
```suggestion
				node{
```

Reply:
I was experimenting with having a `context *yaml.Node` field for a `seqItemNode` to address https://github.com/aws/copilot-cli/pull/4611#discussion_r1132963408. Since `node` wouldn't be the only field in `seqItemNode`, the `node: ` was needed. I guess that change was somehow picked into this branch lol